change default logger namespace to an empty string. fixes #11

derickbailey opened the pull request:
as stated in issue #11, changes default logger namespace to "" instead of "whistlepunk"

brianedgerton commented:
This doesn't seem to pass the tests. Also, would you mind rebasing your branch on master?

derickbailey commented:
d'oh! sorry... fixing now.

derickbailey commented:
ok, i see what happened...

the PR for #8 broke the tests, and @arobson pulled it in before i fixed them. this PR broke because of that. i submitted pr #12 to fix the tests i broke in #8. 

i've rebased this PR against the branch I used for #12 and the tests are passing
